elastic/.env.template

```diff
@@ -0,0 +1,4 @@
+ES_VERSION=6.6.2
+ES_HOST=
```

Comment:
Does FileBeat support CloudID? This would make this a little easier to use and configure:
https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html

Reply:
Filebeat apparently doesn't support CloudID. Filebeat connects to Logstash and Logstash pushes to the Cloud.

Reply:
Filebeat documentation shows it in use here:
https://www.elastic.co/guide/en/beats/filebeat/master/configure-cloud-id.html